zeppelin-web/src/index.html

```diff
@@ -69,6 +69,7 @@
     <!-- endbuild -->
     <link rel="stylesheet" ng-href="assets/styles/looknfeel/{{looknfeel}}.css" />
     <link rel="stylesheet" href="assets/styles/printMode.css" />
+    <link rel="stylesheet" href="//js.arcgis.com/4.0/esri/css/main.css" />
```

Comment:
Any way to have this CSS file locally? It seems it is not inside of angular-map-esri and the map is not rendering well without it.

If the license allows it, it would better to include it in the code `assets/styles/` since some people do not have access to internet when using Apache Zeppelin